lib/loaders/css-extract.js

```diff
@@ -22,13 +23,17 @@ module.exports = {
      */
     prependLoaders(webpackConfig, loaders) {
         if (!webpackConfig.extractCss) {
+
+            const options = {
+                sourceMap: webpackConfig.useSourceMaps,
```

Comment:
#710 has been merged, you can remove this _again_ :D

Reply:
I've done that.